.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - checkout
       - run:
           name: Update npm
-          command: 'sudo npm install -g npm@9.2.0'
+          command: 'sudo npm install -g npm@latest'
```

Comment:
Do we want to be installing the latest version of NPM at all times? If a breaking change were to be introduced, this would break our deployment pipeline no?

Reply:
Node Image 18.16.0-browsers needs build-essential 12.9ubuntu3, curl 7.81.0, docker 20.10.23, docker-compose Docker Compose version v2.16.0 v2.16.0, dockerize v0.8.0, git 2.40.0, java 11.0.18, jq 1.6, node 18.16.0, npm 9.5.1, selenium 3.141.59, ubuntu 22.04.2 LTS, wget 1.21.2, yarn 1.22.19.

i don't think it will break pipeline.